js/bitmex.js

```diff
@@ -970,22 +970,24 @@ module.exports = class bitmex extends Exchange {
          */
```

Comment:
Can you please revert all the other files except this one? All of them are automatically generated from the JS version

Reply:
good to know, reverted.